controllers/keystoneapi_controller.go

```diff
@@ -65,11 +65,6 @@ func (r *KeystoneAPIReconciler) GetKClient() kubernetes.Interface {
 	return r.Kclient
 }
```

Comment:
GetClient, GetKClient and GetScheme could also be removed. its a relikt from before we introduced the helper. Could also do it in a follow up.